### DISCUSSION TITLE
Save Startup Settings

### DISCUSSION THREAD

**Discussion by Marc0ai** [Bing-su/adetailer#629] [Ideas]
Hi, as the title suggests, it would be nice to have a button that keeps the settings saved to be used as default at each startup.
It seems silly, but it could be a very handy thing to have and I also don't think it's that difficult to implement.
👋

**Comment by Woisek** [comment]
I suggested preset savings with ADetailer some time ago, but somehow there seems to be no interest in implementing such a helpful feature ... 🤷‍♂️

**Reply by BoxJockey** [reply]
Yes, I wish for this every time I need to restart the UI.

**Comment by brknsoul** [comment]
There is.

Upon launching A1111 or Forge, you can set up your most common settings (eg: height/width, sampler, steps, etc, etc) including positive and negative prompts, and including extension settings, then go to Settings > Defaults (near the bottom of the settings list), then click View Changes to check what changes will be the defaults, then click Apply.
Next time you start A1111, those new settings will be the defaults
